web/firebase/constants-example.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I don't think you need to add this file. If we're using firebase as a backend, we probably know what firebase constants are. If the goal is to explain how the connection to firebase works for new contributors, a description in the readme would make more sense I think.

Reply:
Ah yes seems redundant. Fixed